api/v1beta1/policyautomation_types.go

```diff
@@ -50,6 +50,8 @@ type AutomationDef struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinLength=1
 	TowerSecret string `json:"secret"`
+	// JobTTL sets the time to live for k8s job object after the playbook run has finished
```

Comment:
Since this is technically user facing when examining the CRDs, let's make this a little more official sounding. :smile: 
```suggestion
	// JobTTL sets the time to live for the Kubernetes AnsibleJob object after the Ansible job run has finished.
```